draft-ietf-quic-transport.md

```diff
@@ -1660,8 +1660,8 @@ on its own.
 The primary defense against amplification attack is verifying that an endpoint
 is able to receive packets at the transport address that it claims.  Address
 validation is performed both during connection establishment (see
-{{validate-handshake}}) and during connection migration (see
-{{migrate-validate}}).
+{{validate-handshake}}) and during connection migration; see
+{{migrate-validate}}.
```

Comment:
```suggestion
{{validate-handshake}}) and during connection migration (see
{{migrate-validate}}).
```
I might leave this one alone, since it's explicitly giving details about each of two things.